tests/src/lib.rs

```diff
@@ -119,14 +119,35 @@ pub fn mk_compiler(
     #[cfg(cargo_profile = "release")]
     let mode = "release";
 
+    let prelink_args = env::var("PRELINK_PASSES").unwrap_or_default();
+    let postlink_args = env::var("POSTLINK_PASSES").unwrap_or_default(); 
+
+    let mut yk_config_args: Vec<&str> = Vec::new();
```

Comment:
This and the next line can probably become `let mut yk_config_args = vec![mode]`.

Reply:
addressed here: 9e4bf0f2470deb6e2fa8cc2fd5aa2fe12d225bff